abdulfataiaka/more_recipes
# #154697734 - A well formed readme for application

#### What does this PR do?

- Adds a well-formed and standard readme content to the `ReadMe.md` file

#### How should this be manually tested?
- Navigate to `https://github.com/abdulfataiaka/more_recipes`
- Scroll down to see the  readme section below

 #### Description of Task to be completed?
- Describes different API endpoint for the application
- States the different technologies used

#### What are the relevant pivotal tracker stories?

- story type: chore
- story id: 154697734
- story title: There should be a well formed readme for app on Github

 ####Any background context you want to provide?

N/A

Screenshots (if appropriate)

N/A
